recipes/spans/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "spans" %}
+{% set version = "0.3.0" %}
+{% set sha256= "d17d36dc240525e97eae26ff68b63b8a9dc6581e4668581f6e189c24ea50f201" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/74/73/2d0192f38a8caad9ab439210ef943825312902cc9280c9b73a196b5be719/Spans-0.3.0.tar.gz
```

Comment:
You should set the name to:

```
{% set name = "Spans" %}
```

(note the capitalization). Then the url would be:

```
url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

as in: 

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18